src/NuGet.Clients/PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -368,6 +415,39 @@ protected void SelectVersion()
 
         public abstract bool IsSolution { get; }
 
+        private string _optionsBlockedMessage;
+
+        public virtual string OptionsBlockedMessage
+        {
+            get
+            {
+                return _optionsBlockedMessage;
+            }
+
+            set
+            {
+                _optionsBlockedMessage = value;
+                OnPropertyChanged(nameof(OptionsBlockedMessage));
+            }
+        }
+
+        private Uri _optionsBlockedUrl;
+
+        public virtual Uri OptionsBlockedUrl
+        {
+            get
+            {
+                return _optionsBlockedUrl;
+            }
+
+            set
+            {
+                _optionsBlockedUrl = value;
+                OnPropertyChanged(nameof(OptionsBlockedUrl));
+                OnPropertyChanged(nameof(OptionsBlockedMessage));
```

Comment:
The two are combined later on, if we update the url the UI should also update the message.